deps/media-playback/media-playback/media.h

```diff
@@ -135,6 +137,7 @@ extern void mp_media_play_pause(mp_media_t *media, bool pause);
 extern int64_t mp_get_current_time(mp_media_t *m);
 extern void mp_media_seek_to(mp_media_t *m, int64_t pos);
 
+
```

Comment:
Remove this unnecessary whitespace change.

Reply:
removed